referrer-policy/generic/common.js

```diff
@@ -0,0 +1,9 @@
+
+// Adds a PRE tag to end of body with propertis of the current page.
+function appendPageInfo(title) {
```

Comment:
It's not entirely clear to me why you append this data to the document. Is this just for manual debugging? The actual tests should be assertion-based and automated.